### PR TITLE
spike: style newsletter signup

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksstyle-newsletter-signup-hashicorp.vercel.app/certifications) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1203510379774620) 🎟️

## 🗒️ What

Spikes out styling the marketo form and newsletter signup block

Screen Shot 2022-12-08 at 2 05 30 PM

### CODE REVIEW

File: src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
+.formWrapper {
+	width: 100%;
+}
+
+.form {
```
Comment (kendallstrautman) [inline src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css:5]: This file is pretty messy, but im not sure another way to style out this marketo form component without these hacks, feels _okay_ as a first spike 🤷🏻

File: src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+		& [data-field-id='Consent_Privacy_Policy__c'] {
+			grid-row: 2 / 3;
+			grid-column: 1 / -1;
+		}
+	}
+}
+
+.dark {
+	& [data-field-id='Email'] input {
+		border-radius: var(--token-form-control-border-radius);
```
Comment (kendallstrautman) [inline src/views/certifications/views/landing/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css:61]: Updated what I immediately saw to match hds, there were a few things that I couldn't match to spec without the styles totally crumbling (standouts are the email input / button height and the checkbox height)

**Comment by kendallstrautman** [comment]
@zchsh feel free to take this into the cert spike if its helpful as a first pass. Also hit me with any feedback and I can clean up! Whatever is most helpful for you 😄

**Comment by zchsh** [comment]
@kendallstrautman This seems awesome, merging it in! 💯 Will definitely reach out if I can use any help during refinement & QA etc 🙇

**Pull request merged**